### PR TITLE
feat: generativeaionvertexai_embedding_batch

### PR DESCRIPTION
**Pull request by gryczj**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [x] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] This pull request is from a branch created directly off of `GoogleCloudPlatform/nodejs-docs-samples`. Not a fork.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by iennae** [comment]
@gemini review

File: ai-platform/snippets/create-batch-embedding.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+async function main(projectId, inputUri, outputUri, jobName) {
+  // [START generativeaionvertexai_embedding_batch]
+  // Imports the aiplatform library
+  const aiplatformLib = require('@google-cloud/aiplatform');
+  const aiplatform = aiplatformLib.protos.google.cloud.aiplatform.v1;
+
+  /**
+   * TODO(developer):  Uncomment/update these variables before running the sample.
+   */
```
[the remark this answers is not in the document]
Comment (iennae) [reply ai-platform/snippets/create-batch-embedding.js:27]: This doesn't follow our style guidelines and would make sample management unwieldy

File: ai-platform/snippets/create-batch-embedding.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  // Optional: URI of the input dataset.
+  // Could be a BigQuery table or a Google Cloud Storage file.
+  // E.g. "gs://[BUCKET]/[DATASET].jsonl" OR "bq://[PROJECT].[DATASET].[TABLE]"
+  // inputUri =
+  //   'gs://cloud-samples-data/generative-ai/embeddings/embeddings_input.jsonl';
+
+  // Optional: URI where the output will be stored.
+  // Could be a BigQuery table or a Google Cloud Storage file.
+  // E.g. "gs://[BUCKET]/[OUTPUT].jsonl" OR "bq://[PROJECT].[DATASET].[TABLE]"
+  // outputUri = 'gs://your_backet/embedding_batch_output';
```
[the remark this answers is not in the document]
Comment (iennae) [reply ai-platform/snippets/create-batch-embedding.js:39]: this is a good catch; @gryczj can you implement the change.

[the remark this answers is not in the document]
Comment (gryczj) [reply ai-platform/snippets/create-batch-embedding.js]: @iennae fixed

File: ai-platform/snippets/test/create-batch-embedding.test.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  const projectId = process.env.CAIP_PROJECT_ID;
+  const storage = new Storage({
+    projectId,
+  });
+  let batchPredictionJobId;
+  let bucket;
+
+  before(async () => {
+    const bucketName = `test-bucket-${uuid()}`;
+    // Create a Google Cloud Storage bucket for UsageReports
```
[the remark this answers is not in the document]
Comment (gryczj) [reply ai-platform/snippets/test/create-batch-embedding.test.js]: @m-strzelczyk fixed

**Pull request merged**